zerver/migrations/0001_initial.py

```diff
@@ -47,8 +53,8 @@ class Migration(migrations.Migration):
 CREATE TRIGGER zerver_message_update_search_tsvector_async
   BEFORE INSERT OR UPDATE OF subject, rendered_content ON zerver_message
   FOR EACH ROW EXECUTE PROCEDURE append_to_fts_update_log();
-"""),
-            ]
+""")
+        zulip_postgres_migrations = [full_text_search_migration]
 
```

Comment:
I'm not sure it needs to be a large task.  Assuming we're OK with the old full-text search still having its implementation present, it seems to me like we need the script to just do a few things:

* Do an ALTER ROLE to update the default search path to the appropriate value
* Run the CREATE INDEX command above

Or am I missing something?